pkg/operation/botanist/component/backupentry/backupentry.go

```diff
@@ -163,6 +135,77 @@ func (b *backupEntry) Wait(ctx context.Context) error {
 	})
 }
 
+// Migrate uses the garden client to deschedule the BackupEntry from its current seed.
+func (b *backupEntry) Migrate(ctx context.Context) error {
+	backupEntry := &gardencorev1beta1.BackupEntry{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      b.values.Name,
+			Namespace: b.values.Namespace,
+		},
+	}
+
+	if err := b.client.Get(ctx, kutil.Key(b.values.Namespace, b.values.Name), backupEntry); err != nil {
+		return client.IgnoreNotFound(err)
```

Comment:
Just to confirm: This returns `nil` when the `BackupEntry` was not found. Is this the expected behaviour?

Reply:
Yes, I assumed that if the `core.BackupEntry` was not found it might have been deleted and therefore the `extensions.BackupEntry` in the Seed cluster should also be deleted.